mailu/templates/front/service-external.yaml

```diff
@@ -19,6 +19,9 @@ spec:
 {{- with .Values.front.externalService }}
   type: {{ .type | default "ClusterIP" }}
   externalTrafficPolicy: {{ .externalTrafficPolicy | default "Local" }}
+  {{- with .ipFamilyPolicy }}
```

Comment:
Why use `with` here and not `if`? 
It would seem that it's not certain that `ipFamilyPolicy` exists.
`with` assumes your variable is present: https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with